aea/agent.py

```diff
@@ -125,7 +125,7 @@ def liveness(self) -> Liveness:
     @property
     def tick(self) -> int:
         """Get the tick."""
-        return self._tick
+        return self._tick  # pragma: no cover
```

Comment:
This should be tested. It's a trivial test, just run the agent and look if the tick is incremented.